app/models/transaction.rb

```diff
@@ -8,7 +8,7 @@ class Transaction < ApplicationRecord
 
 	setup_houid :trx, :houid
 	
-	belongs_to :supporter
+	belongs_to :supporter, required: true
```

Comment:
The shoulda-matchers work slightly different in Rails 5. Even a belongs_to is validated as present, the matcher won't work unless I set required. Don't know why.